### PR TITLE
PKG-1114 update to 1.5.4

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,7 +27,7 @@ requirements:
     - make          # [unix]
     # Require `cmake-no-system` to break circular dependency;
     # the `cmake` package on defaults requires `zstd`.
-    - cmake-no-system
+    - cmake-no-system
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Remove an additional whitespace

**Review by skupr-anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml]: done

**Review by cbouss — COMMENTED** [review]
We can enable xz for Windows too.

From what I see, it seems to me that xz now includes features as highlighted here:
https://cmake.org/cmake/help/v3.0/module/FindLibLZMA.html

If you look at https://anaconda.org/anaconda/xz/5.2.10/download/win-64/xz-5.2.10-h8cc25b3_1.tar.bz2, the functions are there.

**Pull request merged**